diff -ur netatalk-3.0.2-o/etc/afpd/afp_mdns.c netatalk-3.0.2/etc/afpd/afp_mdns.c
--- netatalk-3.0.2-o/etc/afpd/afp_mdns.c	2013-01-21 10:28:44.000000000 -0700
+++ netatalk-3.0.2/etc/afpd/afp_mdns.c	2013-03-19 17:57:03.000000000 -0600
@@ -237,7 +237,7 @@
             LOG(log_info, logtype_afpd, "Registering server '%s' with model '%s'",
                 dsi->bonjourname, obj->options.mimicmodel);
             TXTRecordCreate(&txt_devinfo, 0, NULL);
-            TXTRecordPrintf(&txt_devinfo, "model", obj->options.mimicmodel);
+            TXTRecordPrintf(&txt_devinfo, "model=%s", "%s", obj->options.mimicmodel);
             error = DNSServiceRegister(&svc_refs[svc_ref_count++],
                                        0,               // no flags
                                        0,               // all network interfaces
