diff -ur netatalk-3.0.2-o/configure.ac netatalk-3.0.2/configure.ac
--- netatalk-3.0.2-o/configure.ac	2013-01-21 10:28:44.000000000 -0700
+++ netatalk-3.0.2/configure.ac	2013-03-19 17:44:47.000000000 -0600
@@ -7,7 +7,7 @@
 
 AC_CANONICAL_SYSTEM
 AM_INIT_AUTOMAKE(netatalk, ${NETATALK_VERSION})
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADER([config.h])
 AM_MAINTAINER_MODE([enable])
 
 dnl Checks for programs.
