diff -uNr netatalk-1.4b2+asun2.1.3.orig/bin/nbp/nbplkup.c netatalk-1.4b2+asun2.1.3/bin/nbp/nbplkup.c
--- netatalk-1.4b2+asun2.1.3.orig/bin/nbp/nbplkup.c	Fri May 30 09:15:44 1997
+++ netatalk-1.4b2+asun2.1.3/bin/nbp/nbplkup.c	Sat Mar 20 14:38:57 1999
@@ -132,9 +132,6 @@
 	exit( -1 );
     }
     for ( i = 0; i < c; i++ ) {
-	asciize( nn[ i ].nn_objlen, nn[ i ].nn_obj );
-	asciize( nn[ i ].nn_typelen, nn[ i ].nn_type );
-
 	printf( "%31.*s:%-34.*s %u.%u:%u\n",
 		nn[ i ].nn_objlen, nn[ i ].nn_obj,
 		nn[ i ].nn_typelen, nn[ i ].nn_type,
@@ -143,18 +140,6 @@
 		nn[ i ].nn_sat.sat_port );
     }
 
-}
-
-#include <ctype.h>
-
-asciize( i, p )
-    int		i;
-    char	*p;
-{
-    for ( ; i > 0; p++, i-- ) {
-	if ( !isascii( *p ))
-	    *p = toascii( *p );
-    }
 }
 
 #define BPLEN	48
