--- netatalk-1.5.3.1/aclocal.m4.orig	Sat Mar 30 13:12:05 2002
+++ netatalk-1.5.3.1/aclocal.m4	Sat Mar 30 13:12:27 2002
@@ -4132,7 +4132,7 @@
 		[  --with-nls-dir=PATH     path to NLS files [PKGCONF/nls]],
 		[
 			if test "x$withval" != "x"; then
-				NLSDIR = "$withval"
+				NLSDIR="$withval"
 			fi
 		]
 	)
--- netatalk-1.5.3.1/macros/config-checks.m4.orig	Sat Mar 30 13:50:28 2002
+++ netatalk-1.5.3.1/macros/config-checks.m4	Sat Mar 30 13:50:42 2002
@@ -20,7 +20,7 @@
 		[  --with-nls-dir=PATH     path to NLS files [PKGCONF/nls]],
 		[
 			if test "x$withval" != "x"; then
-				NLSDIR = "$withval"
+				NLSDIR="$withval"
 			fi
 		]
 	)
