--- netatalk-1.5.3.1/configure.in~	Thu Apr 18 22:23:06 2002
+++ netatalk-1.5.3.1/configure.in	Thu Apr 18 22:23:06 2002
@@ -50,8 +50,6 @@
 AC_CHECK_LIB(lwp, main)
 dnl Replace `main' with a function in -ln:
 AC_CHECK_LIB(n, main)
-dnl Replace `main' with a function in -lnsl:
-AC_CHECK_LIB(nsl, main)
 dnl Replace `main' with a function in -lprot:
 AC_CHECK_LIB(prot, main)
 dnl Replace `main' with a function in -lrx:
