--- netatalk-2.0.1/configure.in	2004-10-27 14:20:38.000000000 +0000
+++ netatalk-2.0.1.new/configure.in	2004-11-03 21:56:24.208669640 +0000
@@ -55,8 +55,6 @@
 dnl AC_CHECK_LIB(lwp, main)
 dnl Replace `main' with a function in -ln:
 dnl AC_CHECK_LIB(n, main)
-dnl Replace `main' with a function in -lnsl:
-AC_CHECK_LIB(nsl, main)
 dnl Replace `main' with a function in -lprot:
 dnl AC_CHECK_LIB(prot, main)
 dnl Replace `main' with a function in -lrx:
