--- netatalk-2.0.3/bin/cnid/cnid_index.c~	2005-04-10 12:49:18.000000000 +0000
+++ netatalk-2.0.3/bin/cnid/cnid_index.c	2005-06-18 14:24:51.777539632 +0000
@@ -274,7 +274,7 @@
     DB_BTREE_STAT *sp;
     DB *db = db_table[dbi].db;
 
-    ret = db->stat(db, &sp, 0);
+    ret = db->stat(db, &sp, 0, 0);
 
     if (ret) {
         LOG(log_error, logtype_cnid, "error getting stat infotmation on database: %s", db_strerror(errno));
--- netatalk-2.0.3/etc/cnid_dbd/dbif.c~	2004-12-21 13:36:12.000000000 +0000
+++ netatalk-2.0.3/etc/cnid_dbd/dbif.c	2005-06-18 14:55:40.141545592 +0000
@@ -514,7 +514,7 @@
     DB_BTREE_STAT *sp;
     DB *db = db_table[dbi].db;
 
-    ret = db->stat(db, &sp, 0);
+    ret = db->stat(db, &sp, 0, 0);
 
     if (ret) {
         LOG(log_error, logtype_cnid, "error getting stat infotmation on database: %s", db_strerror(errno));
