--- netatalk-1.5.3.1/configure.in.orig	Sat Mar 30 13:23:33 2002
+++ netatalk-1.5.3.1/configure.in	Sat Mar 30 13:24:24 2002
@@ -234,9 +234,9 @@
 		fi
 		AC_DEFINE_UNQUOTED(SERVERTEXT, "$msg_dir",
 			[path to message dir [[PKGCONF/msg]]])
-		SERVERTEXT="$msg_dir"
-		AC_SUBST(SERVERTEXT)
 )
+SERVERTEXT="$msg_dir"
+AC_SUBST(SERVERTEXT)
 
 dnl Check for optional server location protocol support (used by MacOS X)
 NETATALK_SRVLOC
